packages/application-shell-connectors/package.json

```diff
@@ -18,8 +18,15 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./src/index.js",
-  "module": "./src/index.js",
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "cross-env npm run build:es && npm run build:cjs",
+    "build:watch": "cross-env npm run build:es -- -w",
+    "build:cjs": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f cjs ./src/index.js -o dist/application-shell-connectors.cjs.js",
+    "build:es": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f es ./src/index.js -o dist/application-shell-connectors.es.js"
+  },
+  "main": "./dist/application-shell-connectors.cjs.js",
+  "module": "./dist/application-shell-connectors.es.js",
```

Comment:
Nitpick: can we keep the same order for the keys in all package jsons? E.g. `main` and `module` come after `publishConfig` 😇 🙏

Reply:
Yeah went through and did this. Also added "files"